Makefile

```diff
@@ -132,7 +132,7 @@ endif
 
 .PHONY: format
 format:
-	go install -C internal/devtools github.com/cockroachdb/crlfmt && crlfmt -w -tab 2 ../..
```

Comment:
I was pretty perplexed by this. To me this `../..` seems transparently obviously wrong and it should be `.`. This change fixes CI ([link](https://github.com/cockroachdb/pebble/actions/runs/11222624774/job/31195468780?pr=4012)).

Reply:
Indeed, thanks for the fix. I think there was an intermediate version that used `go run` and the working dir would be `./internal/devtools`.